research/README.MD

```diff
@@ -0,0 +1,41 @@
+# Research Directory
+Research directory will be the place to host various research work from community on Federated learning
+leveraging NVIDIA FLARE. **The code will not be maintained by NVIDIA FLARE team**, but will require Pull Request
+approval process. 
+
+## License
+By providing the code in NVFLARE repository, you will grant the research project in NVIDIA repo to be released under open source license
```

Comment:
"LICENSE file should be added to the folder (if other than Apache? how would we handle other licenses?)"